tools/licensegenerator/src/commandline.h

```diff
@@ -38,6 +38,7 @@ class CommandLine
 			ValueArg<std::string>			date					("d",  "date", "Expiry date, format: day/month/year -> 30/12/2025", false, "", "expiry_date");
 			ValueArg<std::string>			tag						("t",  "tag", "Additional information", false, "", "tag");
 			ValueArg<std::string>			uuid					("u",  "uuid", "Generate unique user id", false, "", "uuid");
+			ValueArg<std::string>			signing_scheme			("s",  "signing_scheme", "Signing scheme", false, "", "signing_scheme");
```

Comment:
Can we provide the options here? 'RSASS_PKCS1v15_SHA1' etc.

Reply:
In progress